<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreReaderClass</class>
 <widget class="QMainWindow" name="ScoreReaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScoreReader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="originLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionFindNote"/>
    <addaction name="actionFineLine"/>
    <addaction name="actionFindMark"/>
    <addaction name="actionExtractMIDI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcess"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open images</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>종료</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionFindNote">
   <property name="text">
    <string>Find a notes</string>
   </property>
  </action>
  <action name="actionFineLine">
   <property name="text">
    <string>Find a lines</string>
   </property>
  </action>
  <action name="actionFindMark">
   <property name="text">
    <string>Find a marks</string>
   </property>
  </action>
  <action name="actionExtractMIDI">
   <property name="text">
    <string>Extract to MIDI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="scorereader.qrc"/>
 </resources>
 <connections/>
</ui>
